kie-spring-boot/kie-spring-boot-samples/keycloak-kie-server-spring-boot-sample/pom.xml

```diff
@@ -130,6 +166,18 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <systemPropertyVariables>
+              <keycloak.http.port>8080</keycloak.http.port>
```

Comment:
That's the original TCP port: it is from the _perspective of the container. From the host's perspective Testcontainers actually exposes this on a random free port_ (obtained with getMappedPort) _This is by design this way, to avoid port collisions that may arise with locally running software or in between parallel test runs._